src/main/scala/io/iohk/praos/domain/BlockchainState.scala

```diff
@@ -0,0 +1,7 @@
+package io.iohk.praos.domain
+
+import akka.util.ByteString
+
+case class BlockchainState(blockchain: Blockchain, maybeHeadBlockHash: Option[ByteString], receivedChains: List[Blockchain])
```

Comment:
Maybe it would be better to rename the `blockchain` field with something that references the fact that it starts from block 1, and so differentiate it from the standard meaning?

Reply:
:+1: